lib/rules/function-allowed-list/README.md

```diff
@@ -73,3 +73,46 @@ a {
     -moz-linear-gradient(45deg, blue, red);
 }
 ```
+
+## Optional secondary options
+
+### `exceptWithoutPropertyFallback`: `["array", "of", "unprefixed", /functions/, "/regex/"]|"function"|"/regex/"|/regex/`
+
+Requires specified functions to have a fallback property declaration before them within the same selector.
+The function must be included in the primary option list to be affected by this option.
```

Comment:
```suggestion
Disallow the matching properties when they are without a property fallback in the same declaration block.

For example, with `["scale", "min", "/max/"]`.
```

For `except*` options we typically write:

> Reverse the primary option for x

[For example](https://stylelint.io/user-guide/rules/alpha-value-notation/#exceptproperties-regex-regex-string).

As the primary option for this rule is an array, let's be specific that its disallowing (rather than allowing) the function.